code-samples/eventing/bookstore-sample-app/frontend/client/components/CommentList.js

```diff
@@ -0,0 +1,13 @@
+import CommentDisplay from './CommentDisplay';
+const CommentList = () => {
+	const comment = {
+		avatar: '/images/avatar.jpg',
+		time: '10:05',
+		text: 'I used this provider to insert a different theme object depending on a person ',
+		emotion: '😃',
```

Comment:
The response for emotion we will get from the backend should be a string, and they can only be:
- Positive
- Neutral
- Negative

So we may need to find a way to deal with that. If we detect the response is "Positive", we will show the happy emoji 😃; If we detect the response is "Neutral", we will show 😐; If we detect the response is "Negative", we will show the angry 😡;